src/Microsoft.DotNet.Interactive/KernelExtensions.cs

```diff
@@ -454,7 +454,7 @@ object ParseValueOption(ArgumentResult argResult)
 
         var fromKernelOption = new Option<string>(
             "--from",
-            "The name of the kernel to get the value from.");
+            LocalizationResources.MagicsFromDescription());
```

Comment:
```suggestion
            LocalizationResources.Magics_share_from_Description());
```

Another example. Since this one is a child option of the `#!share` command, I'd suggest this naming.

Reply:
I am not sure the suggested change.

Methods are already using pascal case and If there are analyzers checking for this naming convention then this suggested change can break a rule.

Reply:
changed